permission_handler_android/android/src/main/java/com/baseflow/permissionhandler/PermissionManager.java

```diff
@@ -481,12 +481,36 @@ private int checkNotificationPermissionStatus(Context context) {
     }
 
     private int checkBluetoothPermissionStatus(Context context) {
-        List<String> names = PermissionUtils.getManifestNames(context, PermissionConstants.PERMISSION_GROUP_BLUETOOTH);
-        boolean missingInManifest = names == null || names.isEmpty();
-        if (missingInManifest) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
```

Comment:
I have just spent some time with this library and getting to know its code and testing it on various Android devices, and here is the result of my research: this change (and PR) seems to be unnecessary, at least with version 10.4.2. 

As you can see here https://github.com/Baseflow/flutter-permission-handler/blob/permission_handler_v10.4.2/permission_handler_android/android/src/main/java/com/baseflow/permissionhandler/PermissionManager.java#L322-L324 the `checkBluetoothPermissionStatus` method edited here is only used if the new permissions are checked on a device with API level < S (31) - in this case the library will simply fall back to checking for the old `Permissions.BLUETOOTH`. If the current device is 31 or higher, the method will not be called and the permissions will be checked by the code starting here: https://github.com/Baseflow/flutter-permission-handler/blob/permission_handler_v10.4.2/permission_handler_android/android/src/main/java/com/baseflow/permissionhandler/PermissionManager.java#L327.

Similarly, `requestPermissions` (not touched in this PR) works correctly - on pre-31 devices it will first determine the status of the permissions (by using `checkBluetoothPermissionStatus` and the same fallback logic as described above) and because `Permissions.BLUETOOTH` is a 'normal' permission, as long as it is in the manifest it will be 'granted' (not having it in the manifest is an error), so requesting it is a no-op - it will not put the permission to `permissionsToRequest` list. On 31 and later devices, it will correctly request the new permissions.

Which means - as long as the Flutter part of you application uses new permissions, it will all work even on pre-31 devices as the library is smart enough to fallback to the old permission if new ones are used on new devices. Tested with both a 33 and 23 device.

As a side note, irrelevant to this PR, the same logic applies to the new `Permissions.POST_NOTIFICATIONS` introduced in API level 33 - if used on a pre-33 device, this library falls back to using the `NotificationManagerCompat` for the check, and 33 and later check the permission - see https://github.com/Baseflow/flutter-permission-handler/blob/permission_handler_v10.4.2/permission_handler_android/android/src/main/java/com/baseflow/permissionhandler/PermissionManager.java#L479-L492.